### DISCUSSION TITLE
Is the VS Code extension usable in a mono repo with multiple teams apps?

### DISCUSSION THREAD

**Discussion by Robstei** [OfficeDev/microsoft-365-agents-toolkit#12060] [Q&A]
I did not found a way to use the extension when opening a root folder that contains multiple applications.

![image](https://github.com/user-attachments/assets/64664598-0dab-429f-86bc-9804bc95e6b1)

![image](https://github.com/user-attachments/assets/c5d57bb8-f2fb-4497-a92d-4aceeac06473)

Is this possible?

**Comment by therealjohn** [comment]
@Robstei Are those all unique apps with their own manifests or are they the implementation logic for the capabilities in a single app manifest?

Either way, Teams Toolkit expects `teamsapp.yml` and `teamsapp.local.yml` to be at the root of the open directory. 

And it only supports a single app flow.
